keyboards/redox/keymaps/jeherve/keymap.c

```diff
@@ -222,38 +242,29 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
 };
 
-#ifdef AUDIO_ENABLE
-float tone_colemak[][2]     = SONG(ZELDA_TREASURE);
-#endif
-
 void persistent_default_layer_set(uint16_t default_layer) {
-	eeconfig_update_default_layer(default_layer);
-	default_layer_set(default_layer);
+    eeconfig_update_default_layer(default_layer);
+    default_layer_set(default_layer);
 }
 
-// Set unicode in Mac OS.
-void eeconfig_init_user(void) {
-  set_unicode_input_mode(UC_OSX);
-};
-
 // Change LED colors depending on the layer.
 uint32_t layer_state_set_user(uint32_t state) {
-	switch (biton32(state)) {
-		case _SYMB:
-			rgblight_setrgb_orange();
-			break;
-		case _NAV:
-			rgblight_setrgb_springgreen();
-			break;
-		case _INTER:
-			rgblight_setrgb_teal();
-			break;
-		case _ADJUST:
-			rgblight_setrgb_red();
-			break;
-		default: //  for any other layers, or the default layer
-			rgblight_setrgb_yellow();
-			break;
-	}
-	return state;
+    switch (biton32(state)) {
```

Comment:
```suggestion
    switch (get_highest_layer(state)) {
```

Reply:
Done in 0c84dc864023d5ecb9f7b064d3ecbfe1841d8d04